site/docs/numbers/hostingNumbers.mdx

```diff
@@ -95,233 +122,179 @@ Content-Type: application/xml; charset=utf-8
+         <ServiceAddress>
+            <HouseNumber>11235</HouseNumber>
+            <StreetName>Back</StreetName>
+            <City>Denver</City>
+            <StateCode>CO</StateCode>
+            <Zip>27541</Zip>
+            <County>Canyon</County>
+         </ServiceAddress>
+      </Subscriber>
+      <LoaAuthorizingPerson>markmacc</LoaAuthorizingPerson>
+      <TelephoneNumbers>
+         <TelephoneNumber>3032281000</TelephoneNumber>
+         <TelephoneNumber>3032280004</TelephoneNumber>
+      </TelephoneNumbers>
+      <ProcessingStatus>PROCESSING</ProcessingStatus>
+      <Errors />
+   </ImportTnOrder>
+</ImportTnOrderResponse>
 ```
 
-</TabItem>
-</Tabs>
+## Fetch Order Status
 
-## Upload Letter of Authorization (LOA)
+Since the Import TN order is asynchronous, Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `OrderStatus`.
+Order processing times can vary and are not guaranteed, so bandwidth does __not__ recommend setting a timeout on waiting for an order to show either `COMPLETE` or `FAILED` status, but instead relying on a webhook from an orders subscription.
+Please follow the [How to setup Notification Webhook](/docs/numbers/webhooks/orderWebhook) guide for more information.
 
-For **completed** orders, Bandwidth requires a completed Subscriber "Letter of Authorization" (LOA) from the phone number user.  The LOA file is used in the case there is a dispute to ensure the phone number had permission to be enabled for hosted messaging for Bandwidth.
+Otherwise, you can make <Highlight color="#00bf8c">GET</Highlight> request to [Fetch Import TN Order status API](../../../../apis/numbers/#tag/Import-Tn/operation/ReadImportTnOrder) endpoint to poll the order status.
 
-You are able to keep the LOA file within your own system, or upload the file to Bandwidth as part of the `importTnOrder` path.
+## Upload Letter of Authorization (LOA)
 
-### Request URL
+For **completed** orders, Bandwidth requires a completed Subscriber "Letter of Authorization" (LOA) from the phone number user.
+The LOA file is used in the case there is a dispute to ensure the phone number had permission to be enabled for hosted messaging for Bandwidth.
 
-<Highlight color="#079CEE">POST</Highlight> <code>https://dashboard.bandwidth.com/api/accounts/{accountId}/importTnOrders/{orderId}/loas</code>
+You are able to keep the LOA file within your own system, or upload the file to Bandwidth as part of the `importTnOrder` path.
+To upload load file, make a <Highlight color="#079CEE">POST</Highlight> request to [Import TN Order loas API](../../../../../apis/numbers/#tag/Import-Tn/operation/CreateImportTnOrderLoas) endpoint.
+This can be done through tools like [Postman](https://github.com/Bandwidth/postman) or in the cURL.
 
-### Request Parameters
+:::info
+The key attribute to a successful upload is to ensure that the headers are set correctly to support the **type** of the file upload.
```

Comment:
Can you elaborate on this?  Which headers are required and describe them here.  The example shows the use of them, but it would be great to describe what's needed.